### PR TITLE
Implement vouch and unvouch component

### PR DESCRIPTION
**Pull request by danielSbastos**
Hey there,

In this PR, the main changes are the vouch and unvouch features. Apart from this, a major refactor on the account API is done, resulting in the removal of the `fetchAccounts.tsx` file, which was no longer needed after I understood better how to fetch, store and parse the accounts directly from the extension, not needing the local storage anymore.  

image
image

### CODE REVIEW

**Comment by danielSbastos** [comment]
[Implement the vouch component](https://trello.com/c/qH9EVdlU/283-implement-the-vouch-component)

File: .eslintrc.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ module.exports = {
     jest: true,
     node: true,
   },
+  overrides: [{
+    files: ["**/*.tsx"],
+    rules: {
+        "react/prop-types": "off"
```
Comment (danielSbastos) [inline .eslintrc.js:28]: This is not needed because typescript itself already forces the type declaration of props.

**Review by laurogripa — CHANGES_REQUESTED** [review]
Vouching is not working properly, `who` address is wrong and KSM values are not in the right format

**Comment by danielSbastos** [comment]
This PR was moved to this one: https://github.com/KappaSigmaMu/kappasigmamu.github.io/pull/103

**Comment by edrd-f** [comment]
Implemented in https://github.com/KappaSigmaMu/kappasigmamu.github.io/pull/103

**Pull request closed without merging**